src/common-models/schedule-data.model.ts

```diff
@@ -68,6 +69,7 @@ export function createEmptyMonthDataModel(
     shifts: freeShifts,
     isAutoGenerated: true,
     shift_types: _.cloneDeep(shift_types),
+    isCorrupted: false,
```

Comment:
Czy `MonthDataModel` na pewno powinien mieć to pole?

Reply:
Właśnie się nad tym zastanawiałem i postawiłem na opcję, że "lepiej miec niż nie mieć" i będzie to łatwiej obsługiwać w razie czego.